src/components/DeleteHouse.js

```diff
@@ -0,0 +1,45 @@
+import React, { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import { useDispatch, useSelector } from 'react-redux';
+import { deleteHouse } from '../redux/houses/housesSlice';
+import '../styling/deleteHouse.css';
+
+const DeleteHouse = () => {
```

Comment:
- [ ] When I try to delete a house using the link in the navigation, I get this:

![image](https://github.com/mercymugambi/frontend_house_reservations_react_app/assets/57421823/115d54a4-f61d-46bb-a9b9-5bcd8aeff2f0)

Kindly, fix this bug so your app works as intended. 👍